src/App.js

```diff
@@ -1,29 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import Calculator from './Components/Calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => (
```

Comment:
- [ ] Please refactor the App function, it should be class-based to match all the project requirements 👇🏽 

> Use class-based components (All components should be class-based).